wd_core.au3

```diff
@@ -885,6 +887,12 @@ Func _WD_ExecuteScript($sSession, $sScript, $sArguments = Default, $bAsync = Def
 		Return SetError(__WD_Error($sFuncName, $iErr, "HTTP status = " & $_WD_HTTPRESULT), $_WD_HTTPRESULT, $sResponse)
 	EndIf
 
+	If $bReturnAsValue Then
+		Local $oJSON = Json_Decode($sResponse)
+		Local $vValue = Json_Get($oJSON, "[value]")
+		Return SetError($_WD_ERROR_Success, 0, $vValue)

```

Comment:
I would prefer to keep a single point of return. Should be fairly simple to accomplish.

Reply:
Hope third commit is fine.